app/tests/integration/statisticSvc.spec.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * @module statisticSvc.spec
+ *
+ * Jest tests for the StatisticService class.
+ * These tests require a configured connection to a Postgresql database.
+ * It will run migrations before the tests, and then purge any test data when completed.
+ *
+ * @see DataService
+ */
+const helper = require('../common/helper');
+const Knex = require('knex');
+const uuidv4 = require('uuid/v4');
+
+const DataConnection = require('../../src/services/dataConn');
+const StatisticsService = require('../../src/services/statisticSvc');
+
+const { countStatisticsByClient, deleteStatisticsByClient } = require('./dataUtils');
+
+helper.logHelper();
+
+const config = require('../../knexfile');
+
+describe('statservice', () => {
+  let knex;
+  let statisticsService;
+  const CLIENT = `unittesting-${new Date().toISOString()}`;
+
+  beforeAll(async () => {
+    knex = Knex(config);
+    await knex.migrate.latest();
+    const dataConnection = new DataConnection();
+    const connectOK = await dataConnection.checkConnection();
+    if (!connectOK) {
+      throw Error('Error initializing dataService');
+    }
+    statisticsService = new StatisticsService();
+  });
+  afterAll(async () => {
+    await deleteStatisticsByClient(CLIENT);
+    return knex.destroy();
+  });
+
+  afterEach(async () => {
+    await deleteStatisticsByClient(CLIENT);
+  });
+
+  it('should return false on initializing data service without knex', async () => {
+    const dataConnection = new DataConnection();
+    dataConnection.knex = undefined;
+    const connectOK = await dataConnection.checkConnection();
+    expect(connectOK).toBeFalsy();
+  });
+
+  it('should not do anything when writing nothing', async () => {
+    await statisticsService.write(null);
```

Comment:
There should be at least one expect in each test. Can probably test to make sure no exceptions were thrown
```expect(async () => await statisticsService.write(null)).not.toThrow()``` or similar.
Src: https://stackoverflow.com/a/49603347

Either that, or test to see that the number of statistics by client is still 0 as nothing was done.

Reply:
done